api/views.py

```diff
@@ -228,6 +230,8 @@ def finish_rent(request):
 
     # set Bike status back to available
     rent.bike.availability_status = "AV"
+    # set new non static bike ID, so for GBFS observers can not track this bike
+    rent.bike.non_static_bike_uuid = uuid.uuid4()
     rent.bike.save()
```

Comment:
what happens if unique constraint fails?

Reply:
if i correctly undestand, if our randomness is OK, then chance of collision is 2^64. So the system might explode every 18,446,744,073,709,551,615 rents

Reply:
now finish rent doesn't fail anymore, just no ratation of uuid in 80558d6